app/views/hosts/puppet/_main_tab_fields.html.erb

```diff
@@ -0,0 +1 @@
+<%= host_puppet_environment_field form %>
```

Comment:
Is there a way to have pagelets run a function instead of rendering a partial that calls a function?